boa3_test/tests/compiler_tests/test_interop/test_storage.py

```diff
@@ -46,40 +49,31 @@ def test_storage_get_mismatched_type(self):
         path = self.get_contract_path('StorageGetMismatchedType.py')
         self.assertCompilerLogs(CompilerError.MismatchedTypes, path)
 
-    def test_storage_put_bytes_key_bytes_value(self):
-        path, _ = self.get_deploy_file_paths('StoragePutBytesKeyBytesValue.py')
-        runner = BoaTestRunner(runner_id=self.method_name())
-
-        invokes = []
-        expected_results = []
+    async def test_storage_put_bytes_key_bytes_value(self):
+        await self.set_up_contract('StoragePutBytesKeyBytesValue.py')
 
         storage_key_1 = b'test1'
         storage_key_2 = b'test2'
         stored_value = b'\x01\x02\x03'
 
-        invokes.append(runner.call_contract(path, 'Main', storage_key_1))
-        expected_results.append(None)
-
-        invokes.append(runner.call_contract(path, 'Main', storage_key_2))
-        expected_results.append(None)
+        result, _ = await self.call('Main', [storage_key_1], return_type=None, signing_accounts=[self.genesis])
+        self.assertEqual(None, result)
```

Comment:
use `self.assertIsNone` instead
same for all `self.assertEqual(None, results)` in the code